db/migrate/20170223114114_lock_seeded_templates.rb

```diff
@@ -0,0 +1,14 @@
+require Rails.root + 'db/seeds.d/07-data.rb'
+require Rails.root + 'db/seeds.d/08-data.rb'
+
+class LockSeededTemplates < ActiveRecord::Migration
+  def up
+    names = (SEEDED_TEMPLATES + SEEDED_PARTITION_TABLES).map { |attrs| attrs[:name] }
+    Template.where(:name => names).update_all(:locked => true)
```

Comment:
On existing db we need to lock all templates, they were seeded unlocked in previous versions. So we find all templates by name that would be locked in new installation and lock them.